packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -108,11 +123,17 @@ export class Accordion extends FoundationElement {
 
     private activeItemChange = (event): void => {
         const selectedItem = event.target as HTMLElement;
+        this.activeid = event.target.getAttribute("id");
         if (this.isSingleExpandMode()) {
             this.resetItems();
             event.target.expanded = true;
+            event.target.setAttribute("aria-disabled", "true");
```

Comment:
We are storing the `event.target` above as `selectedItem` you should be using it here

Reply:
updated